showcase/app/templates/components/accordion.hbs

```diff
@@ -8,157 +8,122 @@
 <Shw::Text::H1>Accordion</Shw::Text::H1>
 
 <section data-test-percy>
-  <Shw::Text::H2>Content</Shw::Text::H2>
-
-  <Shw::Text::H3>Basic</Shw::Text::H3>
-
-  <Shw::Grid {{style gap="2rem"}} @columns={{2}} as |SG|>
-    <SG.Item @label="One item">
-      <Hds::Accordion as |A|>
-        <A.Item>
-          <:toggle>Item one</:toggle>
-          <:content>
-            <Shw::Placeholder @text="generic content" @height="40" />
-          </:content>
-        </A.Item>
-      </Hds::Accordion>
-    </SG.Item>
 
-    <SG.Item @label="Multiple items">
-      <Hds::Accordion as |A|>
-        <A.Item>
-          <:toggle>Item one</:toggle>
-          <:content>
-            <Shw::Placeholder @text="generic content" @height="40" />
-          </:content>
-        </A.Item>
+  <Shw::Text::H2>Type</Shw::Text::H2>
 
-        <A.Item>
-          <:toggle>Item two</:toggle>
-          <:content>
-            <Shw::Placeholder @text="generic content" @height="40" />
-          </:content>
-        </A.Item>
+  {{#each @model.TYPES as |type index|}}
+    <Shw::Text::H3>{{capitalize type}}</Shw::Text::H3>
 
-        <A.Item>
-          <:toggle>Item Three</:toggle>
-          <:content>
-            <Shw::Placeholder @text="generic content" @height="40" />
-          </:content>
-        </A.Item>
-      </Hds::Accordion>
-    </SG.Item>
-  </Shw::Grid>
+    <Shw::Grid {{style gap="2rem"}} @columns={{2}} as |SG|>
+      <SG.Item @label="One item">
+        <Hds::Accordion @type={{type}} as |A|>
+          <A.Item>
+            <:toggle>Item one</:toggle>
+            <:content>
+              <Shw::Placeholder @text="generic content" @height="40" />
+            </:content>
+          </A.Item>
+        </Hds::Accordion>
+      </SG.Item>
+
+      <SG.Item @label="Multiple items">
+        <Hds::Accordion @type={{type}} as |A|>
+          <A.Item>
+            <:toggle>Item one</:toggle>
+            <:content>
+              <Shw::Placeholder @text="generic content" @height="40" />
+            </:content>
+          </A.Item>
+
+          <A.Item @isStatic={{true}}>
+            <:toggle>Item two</:toggle>
+            <:content>
+              <Shw::Placeholder @text="generic content" @height="40" />
+            </:content>
+          </A.Item>
+
+          <A.Item @containsInteractive={{true}}>
+            <:toggle>Item three</:toggle>
+            <:content>
+              <Shw::Placeholder @text="generic content" @height="40" />
+            </:content>
+          </A.Item>
+        </Hds::Accordion>
+      </SG.Item>
+    </Shw::Grid>
 
-  <Shw::Divider @level={{2}} />
+    {{#if (lt index 1)}}
+      <Shw::Divider @level={{2}} />
+    {{/if}}
+  {{/each}}
 
-  <Shw::Text::H3>Complex</Shw::Text::H3>
+  <Shw::Divider />
 
-  <Shw::Text::H4>With nested Accordions</Shw::Text::H4>
```

Comment:
How about adding an example of a "Card" Accordion nested in a "Flush" one? (And vice versa.) I think that could be a legitimate use vase to better visually distinguish the nested vs. non-nested Accordion so would be useful to see in the Showcase.

Reply:
Yeah, we can mix different types as well. The guidance will not recommend this, but it's worth testing these edge cases.